fuel-poa-coordinator/tests/test_trigger.rs

```diff
@@ -114,15 +101,16 @@ impl BlockProducer for MockBlockProducer {
     }
 }
 
+// TODO: The same code is in the `adapters::transaction_selector::select_transactions`
```

Comment:
what about this todo, should we remove this function in favor of the one in adapters or vice versa ?
not clear from the comment

Reply:
It is a problem that I can't say the right place with the current codebase=D
`TxPool` should do sorting inside, and the block producer should expect sorted transactions from it. This TODO highlights that we need to implement this logic to remove this TODO=)